ddpui/settings.py

```diff
@@ -67,8 +68,11 @@
     "ddpui",
     "django_prometheus",
     "django_extensions",
+    "flags",
 ]
 
+FLAGS = {"AIRBYTE_PROFILE": []}
+
```

Comment:
can we add a comment here what is the purpose of this flag ?
Or if we can add description to a flag via the library which can be shown to the user on the admin panel.

Reply:
Done.